backend/common/src/main/java/org/eclipse/sw360/cyclonedx/CycloneDxBOMImporter.java

```diff
@@ -109,22 +112,21 @@ public class CycloneDxBOMImporter {
     private static final String PROJECT_NAME = "projectName";
     private static final String REDIRECTED_VCS = "redirectedVCS";
     private static final boolean IS_PACKAGE_PORTLET_ENABLED = SW360Constants.IS_PACKAGE_PORTLET_ENABLED;
-    private static final Predicate<ExternalReference.Type> typeFilter = type -> ExternalReference.Type.VCS.equals(type);
-
-    private Set<String> redirectedUrls = new HashSet<>();
+    private static final Predicate<ExternalReference.Type> typeFilter = Type.VCS::equals;
 
     private final ProjectDatabaseHandler projectDatabaseHandler;
     private final ComponentDatabaseHandler componentDatabaseHandler;
     private final PackageDatabaseHandler packageDatabaseHandler;
     private final User user;
     private final AttachmentConnector attachmentConnector;
+    private final RepositoryURL repositoryURL;
 
     // Map of supported hosts and base URL formats
     private static final Map<String, String> VCS_HOSTS = Map.of(
```

Comment:
I'd prefer to call the variable as documented above like `KNOWN_VCS_HOSTS` or `SUPPORTED_VCS_HOSTS`.

Reply:
How is this different from `SW360Constants.VCS_HOSTS`? They have same name and one is read from properties and other is defined here. It is confusing :confused: